app/controllers/katello/concerns/api/v2/hostgroups_controller_extensions.rb

```diff
@@ -18,7 +18,6 @@ module ApiPieExtensions
 
       included do
         include ApiPieExtensions
-
```

Comment:
This might sound picky but it's better to not introduce unnecessary/unrelated changes. The space that was here should be put back :)